src/FlagTypes.ts

```diff
@@ -110,7 +110,7 @@ export const flagCategories: FlagCategory[] = [
                 DisplayName: 'Me too',
                 ReportType: 'AnswerNotAnAnswer',
                 GetComment: () => 'Please don\'t add *"Me too"* as answers. It doesn\'t actually provide an answer to the question. ' +
-                    `If you have a different but related question, then [ask](//${window.location.hostname}$/questions/ask) it ` +
```

Comment:
You can just drop the second `$` here.

Reply:
None of the other links use that syntax, however. I was aiming for consistancy